Apps/Gate/gate.c

```diff
@@ -95,8 +96,7 @@ void Gate_Loop(void)
                 {
                     // This is the first time we perform a convertion
                     // Evaluate the time needed to convert all the data of this configuration and update refresh rate
-                    uint32_t execution_time = ((Luos_GetSystick() - last_time) * 2) + 2;
-                    update_time             = TimeOD_TimeFrom_ms(execution_time);
+                    update_time = (float)RoutingTB_BigestID() * 0.01;
```

Comment:
10 ms for each services seems really eavy. what about 1ms?

Reply:
I changed this according to your proposal.

Reply:
Force merge as N. Rabault request changes has been implemented